tests/conftest.py

```diff
@@ -256,7 +257,7 @@ def database_name():
                 params=[(True, True, False), (True, False, False), (False, True, False),
                         (False, False, False), (True, True, True)],
                 ids=["SSL and Auth", "SSL", "Auth", "Normal", "SSL and Auth with not self signed certificate"])
-async def server_multi(inmanta_config, postgres_db, database_name, request):
+async def server_multi(event_loop, inmanta_config, postgres_db, database_name, request):
```

Comment:
Why is the event_loop fixure required?

Reply:
to ensure proper ordering, not sure if it is really required, but can't hurt